### ISSUE TITLE
Paddle-gpu 1.6.1 中label_smooth有bug，使用label_smooth时模型loss很大而且不断上升，去掉后loss正常下降

### ISSUE THREAD

**Issue by Weili-NLP** [PaddlePaddle/Paddle#21572]
1）PaddlePaddle版本：1.6.1
    2）GPU：Tesla V100, CUDA 9.2, CuDNN 7.6
    3）系统环境：CentOS 7, Python 3.6.9
- 训练信息
    1）单机多卡
    2）正常
    3）Operator信息
- 复现信息：
`label = layers.label_smooth(label=layers.one_hot(input=label, depth=vocab_size),
                                    epsilon=label_smooth_eps)`
`cost = layers.softmax_with_cross_entropy(
        logits=predict,
        label=label,
        soft_label=True if label_smooth_eps else False)`

- 问题描述：使用label_smooth时模型loss很大而且不断上升，去掉label_smooth后loss正常下降

**Comment by wanghaoshuang** [comment]
请问您的模型和数据是？

**Comment by Weili-NLP** [comment]
> 请问您的模型和数据是？

基于transformer结构的文本生成模型，数据是文本摘要数据集

**Comment by parap1uie-s** [comment]
label_smooth本来就是一个抗过拟合的方法吧，参数给得过高也会无法正常拟合？
